hosted/config/database.yml

```diff
@@ -57,7 +57,7 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: support_app_test
+  url: <%= ENV.fetch('DATABASE_URL', 'postgresql://support_app_test:support_app_test@localhost:5432/support_app_test') %>
```

Comment:
This doesn't feel particularly good to me; as it moves us off the straight-and-narrow of Rails. 

What if instead of using DATABASE_URL; we configure the CI container that runs the tests [to set the PGHOST, PGUSER, etc. environment variables](https://www.postgresql.org/docs/9.1/libpq-envars.html) so that we don't have to change how people's development environment works in order for CI to work?